daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -502,11 +502,11 @@
             </xs:documentation>
           </xs:annotation>
         </xs:element>
-        <xs:element name="tdmlImplementation" type="xs:string" default="daffodil" minOccurs="0">
+        <xs:element name="tdmlImplementation" type="daf:TunableTDMLImplementation" default="daffodil" minOccurs="0">
```

Comment:
Does this logic mean that you can't override a daffodilC tunable if you want to test with runtime1? I'm not sure how likely that is, it's feels like a sign of things being over complicated.

Feels complex to me. Seems like it would be easier to just specify two different Runners with different implementations, and using runOneTest on the appropriate tests using the appropriate runners. You could even add the "implementation" attribute to the parser/unparserTestCase to ensure those tests never run with the wrong implementation, and sort of document which implementation the test should work with.